rpcs3/rpcs3qt/settings_dialog.cpp

```diff
@@ -961,6 +962,13 @@ settings_dialog::settings_dialog(std::shared_ptr<gui_settings> guiSettings, std:
 	xemu_settings->EnhanceCheckBox(ui->spuDebug, emu_settings::SPUDebug);
 	SubscribeTooltip(ui->spuDebug, json_debug["spuDebug"].toString());
 
+	// Comboboxes
+
+	xemu_settings->EnhanceComboBox(ui->maxLLVMThreads, emu_settings::MaxLLVMThreads, true);
+	SubscribeTooltip(ui->maxLLVMThreads, json_debug["maxLLVMThreads"].toString());
+	ui->maxLLVMThreads->setItemText(ui->maxLLVMThreads->findData("0"), tr("All (%1)").arg(QThread::idealThreadCount()));
+	ui->maxLLVMThreads->setMaxCount(QThread::idealThreadCount());
```

Comment:
Do you think using `QThread::idealThreadCount()` is better in this context than the function `std::thread::hardware_concurrency()` used in rpcs3/Emu/Cell/PPUThread.cpp? Can't we rely on one unique function to count the maximum number of threads?